tests/unit/misc/test_ipc.py

```diff
@@ -309,7 +309,8 @@ def test_permissions_posix(self, ipc_server):
         sockfile = ipc_server._server.fullServerName()
         sockdir = pathlib.Path(sockfile).parent
```

Comment:
If you move `sockfile_path = pathlib.Path(sockfile)` up a bit, you can use 

```suggestion
        sockdir = sockfile_path.parent
```

here as well instead of having to duplicate it.

Or I guess you could even do

```python
sockfile_path = pathlib.Path(ipc_server._server.fullServerName())
```

like below.